src/util/Icon.js

```diff
@@ -0,0 +1,56 @@
+import ObjectUtil from './Object.js';
+
+/**
+ * Icon related utility methods.
+ */
+const IconUtil = {
+
+  /**
+   * Import custom icons from 'app/icons'. Those icons will be included in an
+   * object ready to be injected inside Vuetify configuration options object under
+   * the icons['values'] property.
+   *
+   * @returns {Object} An object containing the imported icons.
+   *                   Key is the icon name, value contains the icon.
+   */
+  importIcons () {
+    const icons = {};
+
+    try {
+      const pathIcons = IconUtil.importPathIcons();
+      ObjectUtil.mergeDeep(icons, pathIcons);
+    } catch (e) {
```

Comment:
Maybe add a warning for developers in the `catch` block?

Reply:
Warning was added as suggested